lstchain/io/__init__.py

```diff
@@ -1,4 +1,4 @@
-from .containers import DL1ParametersContainer
+from .lstcontainers import DL1ParametersContainer
 from .config import read_configuration_file
 
```

Comment:
You probably want the new modules imported as well, no?
(especially LSTEventSource for users)